src/components/Swap/Settings/MaxSlippageSelect.tsx

```diff
@@ -190,7 +193,7 @@ export default function MaxSlippageSelect() {
                 size={Math.max(maxSlippageInput.length, 4)}
                 value={maxSlippageInput}
                 onChange={(input) => processInput(input)}
-                placeholder={'0.10'}
+                placeholder={allowedSlippage.allowed.toFixed(2)}
```

Comment:
we should show the actual "auto" value we're using here, which can change based on the current trade